proto/servicemanager/v3/servicemanager.proto

```diff
@@ -0,0 +1,297 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package servicemanager.v3;
+
+/***********************************************************************************************************************
+ * SMService
+ **********************************************************************************************************************/
+
+service SMService {
+    rpc RegisterSM(stream SMNotifications) returns (stream CMRequests) {}
+}
+
+message CMRequests {
+    oneof CMRequest {
+        GetUnitConfigStatus get_unit_config_status        = 1;
+        CheckUnitConfig check_unit_config                 = 2;
+        SetUnitConfig set_unit_config                     = 3;
+        RunInstances run_instances                        = 4;
+        InstanceStateAcceptance instance_state_acceptance = 5;
+        SetInstanceState set_instance_state               = 6;
+        OverrideEnvVars override_env_vars                 = 7;
+        SystemLogRequest system_log_request               = 8;
+        InstanceLogRequest instance_log_request           = 9;
+        InstanceCashLogRequest instance_cash_log_request  = 10;
+    }
+}
+
+message GetUnitConfigStatus{}
+
+message CheckUnitConfig{
+    string unit_config = 1;
+}
+
+message SetUnitConfig{
+    string unit_config = 1;
+}
+
+message RunInstances {
+    repeated ServiceInfo services   = 1;
+    repeated LayerInfo layers       = 2;
+    repeated InstanceInfo instances = 3; 
+}
+
+message ServiceInfo {
+    string url            = 1;
+    string service_id     = 2;
+    string provider_id    = 3;
+    uint64 aos_version    = 4;
```

Comment:
Move aos_version, vendor_version, and description into separate struct `VersionInfo`.

Reply:
done